pages/sending/windows.rst

```diff
@@ -0,0 +1,69 @@
+***********************
+Ingest Windows eventlog
+***********************
+
+
+Some allow to send Windows event log via Syslog, others have a proprietary protocol implemented. Graylog can work with those that use Syslog as transport or that speak GELF. One collector that should be named is the `NXLog community edition <https://nxlog.co/products/nxlog-community-edition>`__ that can read the windows event log and forward that to Graylog via GELF. 
```

Comment:
> Some allow to send Windows event log via Syslog

What is the 'some' referring to, here? Agents/collectors? Should it read like:

`Some agents allow sending Windows event logs via Syslog, [...]`